### DISCUSSION TITLE
Quake window, key action, non-default profile?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#16845] [General]
"wtp", the preview WT executable, is known to my shell via the "App Paths" registry key.  And TCC32 is a non-default profile.  

`wtp -w _quake -p TCC32` works to open the quake window with a non-default profile.

Can I do the same with a key-action?  I tried adding a "profile" property to a sample globalSummon action; it was ignored and removed.  I got the default profile.

**Comment by zadjii-msft** [comment]
Kinda sounds like you're looking for #12135?

**Comment by vefatica** [comment]
I don't think so.  I want to press a key combination and have the quake window open with the profile of my choice ... like this (below), but with a named profile.

```
        {
            "command": 
            {
                "action": "globalSummon",
                "desktop": "toCurrent",
                "monitor": "toMouse",
                "name": "_quake",
                "toggleVisibility": true
            },
            "keys": "ctrl+shift+q"
        },
```

**Comment by vefatica** [comment]
No further response?  Does that mean you can't do it with a key action?

**Reply by DHowett** [reply]
Having no further responses could also mean that the team is busy. :)
